### ISSUE TITLE
Quick Menu Location

### ISSUE THREAD

**Issue by shaund** [Belphemur/SoundSwitch#1552]
### What workfow do you want to improve?

Can you add an option to display the quick menu at a fixed location rather than at mouse cursor?

### How to make it better?

n/a	

### Any alternative?

n/a

**Comment by DovieW** [comment]
i agree. i use a hot key to switch devices and it feels jarring that the menu shows up in a seemingly unpredicatble location
